spec/controllers/stories/bulk_create_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Stories::BulkCreateController, type: :controller do
+  let(:project) { create :project }
+  let(:story_1) { attributes_for :story, project: project }
+  let(:story_2) { attributes_for :story, project: project }
+  let(:post_params) do
+    {
+      project_id: project.slug,
+      stories: [story_1, story_2]
+    }
+  end
+
+  context 'create stories to the project' do
+    before do
+      ENV['X_API_TOKEN'] = 'token'
+      headers = { 'HTTP_X_API_TOKEN': ENV['X_API_TOKEN'] }
+      request.headers.merge headers
+    end
+
+    subject { post :create, params: post_params }
+
+    it 'should return created status' do
+      expect(subject).to have_http_status :created
+    end
+
+    context 'when the title is missing' do
+      let(:post_params) do
+        {
+          project_id: project.slug,
+          stories: [{}]
+        }
+      end
+
+      subject { post :create, params: post_params }
+
+      it 'should deny the creation of the stories' do
+        expect{ subject }.to raise_error ActionController::ParameterMissing
+      end
+    end
+  end
+
+  context 'deny when headers is missing' do
```

Comment:
```suggestion
  context 'when headers is missing' do
```

Reply:
Also, in this case it's implicit if it's failing because `ENV['X_API_TOKEN']` is missing or because the header is not passed. Can  we have a test for each of these cases?